docs/en/Installation.md

```diff
@@ -1,40 +1,49 @@
-##Installation
-Either run ```composer require axyr/silverstripe-ideannotator --dev```
+# Installation
 
-Or add ```axyr/silverstripe-ideannotator: "dev-master"``` to `require-dev` in your composer.json file
+Install with composer:
 
-Or download and add it to your root directory.
+```
+composer require --dev silverleague/ideannotator
+```
+
+If you know the version you want to use already, you can also add this to `composer.json`:
+
+```
+"require-dev": {
+    "silverleague/ideannotator": "^2.0"
+}
+```
 
+## Configuration
 
-##Config
-This module is disabled by default and I recommend to only enable this module in your local development environment, since this module changes the file content of the Dataobject and DataExtension classes.
+This module is disabled by default and it is recommended to only enable this module in your local development environment, since this module changes the file content of the `DataObject` and `DataExtension` classes.
 
-You can do this, by using something like this in your mysite/_config.php :
+You can do this by using something like this in your `mysite/_config.php`:
 
 ```php
-if($_SERVER['HTTP_HOST'] == 'mysite.local.dev') {
+if ($_SERVER['HTTP_HOST'] == 'mysite.local.dev') {
```

Comment:
Could we remove this too, replacing it with a `environment.only: dev` yml config? I want to steer away from php stuff when it comes to configs